.tekton/windows-machine-config-operator-bundle-release-4-17-pull-request.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     pipelinesascode.tekton.dev/on-cel-expression: |
       event == "pull_request"
       && target_branch == "release-4.17"
-      && "build/bundle-konflux.Dockerfile".pathChanged()
+      && ( "Containerfile.bundle".pathChanged() || "bundle/***".pathChanged() )
```

Comment:
Please pick up the changes from both commits of https://github.com/openshift/windows-machine-config-operator/pull/2689